README.md

```diff
@@ -279,6 +284,86 @@ We have a small collection of algorithms, one for every day of the month. Scroll
   - [More on the Caesar Cipher](http://practicalcryptography.com/ciphers/caesar-cipher)
 
 
+### **December 15 - Pascal's Triangle**
+- **Problem**
+  - Pascal's Triangle is a triangle(represented as a 2-D array) in which the ends of the rows are 1. Each of the other values is equal to the sum of the two nearest numbers in the row above.
+  - The pascal’s triangle is given as:
+  ```
+	                    1
+                  1          1
+             1          2        1
+         1         3         3      1
+    1       4          6         4     1
+  ```
+ - Up To any number of rows. The task here is to generate a pascal’s triangle for n rows and using this find `(x+y)^n`
```

Comment:
Needs one more level of indentation.

Reply:
What am I supposed to intend here?

Reply:
The "Up to any number" line. It's on the same level as the headings.

Reply:
Sorry, just noticed.